app/src/js/drawable/2d/label2d_handler.ts

```diff
@@ -172,6 +172,71 @@ export class Label2DHandler {
         // unlinking
         this.unlinkLabels()
         break
+      case Key.ARROW_UP:
+        if (Session.label2dList.selectedLabels.length === 1) {
+          const label = Session.label2dList.selectedLabels[0]
```

Comment:
Please put those code segments into separate functions for modularity and readability

Reply:
Ok, I moved them. Should be cleaner now